principal_travel/templates/travel/trip_list.html

```diff
@@ -26,6 +26,35 @@ <h4 class="panel-title">
   </div>
 </div>
 
+<h1>Filters: </h1>
```

Comment:
The Filters heading shouldn't be larger than the Itineraries heading. I'd make the filters heading an <h4>